src/main/java/frc/robot/subsystems/VisionSubsystem.java

```diff
@@ -180,8 +182,10 @@ private void process(
           PhotonUtils.getDistanceToPose(
               FieldPose,
               fieldLayout.getTagPose(result.getBestTarget().getFiducialId()).get().toPose2d());
-      if (Distance > 2.0) return;
-      aprilTagsHelper.addVisionMeasurement(FieldPose, TimestampSeconds, STANDARD_DEVS);
+      aprilTagsHelper.addVisionMeasurement(
+          FieldPose,
+          TimestampSeconds,
+          STANDARD_DEVS.plus(DISTANCE_SC_STANDARD_DEVS).times(Distance));
```

Comment:
@KangarooKoala @Yumnah2 @koolpoolo  This changes things so that at 1m the StdDev is now 0.2, 0.2, 26 and at 2m it's now 0.4, 0.4, 52, which is very different from the 0.1, 0.1, 20 it used to be at 1m and 2m. Due to the method chaining the StdDev at 3m is 0.6, 0.6, 78. You might consider only multiplying the DISTANCE_SC_STANDARD_DEVS by the distance (instead of the sum), and only multiplying by the distance over a bottom limit (1m or 2m by doing something like `.times(Math.max(0,Distance-1))`